modules/mono/editor/GodotTools/GodotTools/Build/NuGetUtils.cs

```diff
@@ -49,7 +49,7 @@ private static void AddFallbackFolderToNuGetConfig(string nuGetConfigPath, strin
             {
                 // Check that the root node is the expected one
                 if (rootNode.Name != nuGetConfigRootName)
-                    throw new Exception("Invalid root Xml node for NuGet.Config. " +
+                    throw new InvalidOperationException("Invalid root Xml node for NuGet.Config. " +
```

Comment:
Maybe `FormatException` or `FileFormatException`?

Reply:
I changed it to `FormatException` because `FileFormatException` has been forwarded to the `System.IO.Packaging` assembly.